e2e/node_delete_test.go

```diff
@@ -0,0 +1,137 @@
+package e2e
+
+import (
+	_ "embed"
+	"errors"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/topolvm/topolvm"
+	topolvmv1 "github.com/topolvm/topolvm/api/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const nsNodeDeleteTest = "node-delete-test"
+
+//go:embed testdata/node_delete/statefulset-template.yaml
+var statefulSetTemplateYAML string
+
+func testNodeDelete() {
+
+	BeforeEach(func() {
+		// Skip because this test require multiple nodes but there is just one node in daemonset lvmd test environment.
+		skipIfDaemonsetLvmd()
+
+		createNamespace(nsNodeDeleteTest)
+	})
+
+	AfterEach(func() {
+		skipIfDaemonsetLvmd()
```

Comment:
`skipIfDaemonsetLvmd` is called two times, in `BeforeEach` and `AfterEach`.
Is it needed? I guess we can delete one of them.

Reply:
I think you are right. I'll check it. thanks.